android/src/main/java/com/bluechilli/flutteruploader/MethodCallHandlerImpl.java

```diff
@@ -112,10 +113,19 @@ private void enqueue(MethodCall call, MethodChannel.Result result) {
     for (Map<String, String> file : files) {
       items.add(FileItem.fromJson(file));
     }
-
     WorkRequest request =
         buildRequest(
-            new UploadTask(url, method, items, headers, parameters, connectionTimeout, false, tag));
+            new UploadTask(
+                url,
+                method,
+                items,
+                headers,
+                parameters,
+                connectionTimeout,
+                false,
+                tag,
+                allowCellular != null ? allowCellular : true));
```

Comment:
`true` could be a documented constant

Reply:
You mean like `const ALLOW_CELLULAR = true;`?